cloud/pod_util.go

```diff
@@ -95,6 +100,22 @@ func ExportPodResources(info pod.ResourceInfo) cocoa.ECSPodResources {
 	return *res
 }
 
+// ExportPodContainerDef exports the ECS pod container definition into the equivalent cocoa.ECSContainerDefintion.
+func ExportPodContainerDef(opts pod.TaskContainerCreationOptions, secretIDs []string) cocoa.ECSContainerDefinition {
+	return *cocoa.NewECSContainerDefinition().
+		SetImage(opts.Image).
+		SetMemoryMB(opts.MemoryMB).
+		SetCPU(opts.CPU).
+		SetEnvironmentVariables(toEnvVars(opts.EnvVars, opts.EnvSecrets, secretIDs))
+}
+
+// ExportPodTaskDef exports the ECS pod resources into cocoa.ECSTaskDefinition.
+func ExportPodTaskDef(resources pod.ResourceInfo) cocoa.ECSTaskDefinition {
```

Comment:
I don't think you need this for now since it only applies to `CreatePodFromExistingDefinition` (which we're not going to use for now). I would leave it out of this PR.

Reply:
Sure, I'll take it out